packages/@expo/cli/src/export/exportStaticAsync.ts

```diff
@@ -390,13 +391,19 @@ export async function exportApiRoutesStandaloneAsync(
     outputDir,
     files = new Map(),
     platform,
+    environment,
   }: {
     outputDir: string;
     files?: ExportAssetMap;
     platform: string;
+    environment?: MetroEnvironment;
   }
 ) {
-  const { serverManifest } = await devServer.getServerManifestAsync();
+  const { serverManifest } = await devServer.getServerManifestAsync({ environment });
+
+  // Clear HTML routes during standalone API route export.
+  serverManifest.htmlRoutes = [];
+  serverManifest.notFoundRoutes = [];
```

Comment:
Need to move the `notFoundRoutes` reset to `exportApiRoutesAsync` below. We're still getting it in the export otherwise.